user_management/models/mixins.py

```diff
@@ -55,6 +56,13 @@ class Meta:
         abstract = True
 
 
+class TimeZoneMixin(models.Model):
+    timezone = models.CharField(max_length=255, default=settings.TIME_ZONE)
```

Comment:
Do you think it would be safer to define another setting? It looks like `TIME_ZONE` could be set to `None` see https://docs.djangoproject.com/en/1.8/ref/settings/#time-zone